data_handler/handlers/order_books/ekubo/histogram.py

```diff
@@ -0,0 +1,140 @@
+import argparse
+import matplotlib.pyplot as plt
+from typing import List
+
+from web_app.order_books.ekubo.main import EkuboOrderBook
+
+TOKEN_A = "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7"  # ETH
+TOKEN_B = "0x53c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8"  # USDC
+
+
+def fetch_order_book_and_current_price(token_a: str, token_b: str) -> tuple:
```

Comment:
Pls add types of individual items in the tuple. Holds for all other occurrences, and also for `dict`, `list`, `set`, etc.